t/21_sles4sap_azure_cli.t

```diff
@@ -649,4 +649,67 @@ subtest '[az_network_peering_delete]' => sub {
     ok((any { /az network vnet peering delete/ } @calls), 'Correct composition of the main command');
 };
 
+
+subtest '[az_disk_create] Create disk by cloning' => sub {
+    my $azcli = Test::MockModule->new('sles4sap::azure_cli', no_auto => 1);
+    my @calls;
+    $azcli->redefine(assert_script_run => sub { @calls = $_[0]; return; });
+
+    az_disk_create(resource_group => 'Pa_a_Pi', name => 'Od_Kuka_do_Kuka', source => 'Harvepino');
+    note("CMD: " . join(' ', @calls));
```

Comment:
Could you eventually print it as in all the other tests `note("\n --> " . join("\n --> ", @calls));` ?

Reply:
Fixed including other occasions.